### DISCUSSION TITLE
Please give me some Music Recommendations!!!

### DISCUSSION THREAD

**Discussion by kaifanli1013** [NAIST-SE/AttractingContributors#826] [General]
Hello everyones! 

I am a music lover.
My favourite singer/band in jp are Aimer and official髭男dism.
And i would like to listen some kind of r&b, jazz, city-pop music recently.

If you have your favourite music to recommend, pls let me know!!!
Anyone is ok!!!

If you give me a PR, I will also reply quickly.

[Music-collection](https://github.com/kaifanli1013/AttractingContributors-Music.git)

**Comment by KKKKKXY** [comment]
I added my recommand song into your list. (githubid: KKKKKXY)

**Comment by oto10** [comment]
I added my favorite music to your list.
Could you please add your favorite music to my README.md?
https://github.com/oto10/AttractingContributors-your-favorite-music
Thank you in advance.

**Comment by k-mizoguch1** [comment]
plz https://github.com/kzu000/Food-what-do-you-like
